src/views/AddItem.jsx

```diff
@@ -62,71 +66,115 @@ export function AddItem({ data, listToken }) {
-							checked={nextPurchase === SOON}
-							onChange={(e) => setNextPurchase(SOON)}
-						/>
-						Soon
-					</label>
-
-					<label htmlFor="kindaSoon">
-						<input
-							type="radio"
-							id="kindaSoon"
-							name="purchase-frequency"
-							checked={nextPurchase === KINDA_SOON}
-							onChange={(e) => setNextPurchase(KINDA_SOON)}
-						/>
-						Kinda Soon
-					</label>
-
-					<label htmlFor="notSoon">
-						<input
-							type="radio"
-							id="notSoon"
-							name="purchase-frequency"
-							checked={nextPurchase === NOT_SOON}
-							onChange={(e) => setNextPurchase(NOT_SOON)}
-						/>
-						Not Soon
-					</label>
-				</fieldset>
-
-				<button type="submit">Add Item</button>
-			</form>
 
-			{submissionConfirmation && <p>{submissionConfirmation}</p>}
+					<fieldset
+						name="buyAgain"
+						className="grid border border-white px-10 py-5 gap-2 my-5 shadow-lg"
+					>
+						<legend htmlFor="buyAgain" className="text-center w-44 font-bold">
+							How soon will you need to buy this again?
+						</legend>
+						<label htmlFor="soon" className={radioLabelStyle}>
+							<input
+								type="radio"
+								id="soon"
+								name="purchase-frequency"
+								checked={nextPurchase === SOON}
+								onChange={(e) => setNextPurchase(SOON)}
+								className="hidden"
+							/>
+							{nextPurchase === SOON ? (
+								<FontAwesomeIcon icon={faCircleDot} className={radioStyle} />
+							) : (
+								<FontAwesomeIcon icon={faCircle} className={radioStyle} />
+							)}
+							Soon
+						</label>
+
+						<label htmlFor="kindaSoon" className={radioLabelStyle}>
+							<input
+								type="radio"
+								id="kindaSoon"
+								name="purchase-frequency"
+								checked={nextPurchase === KINDA_SOON}
+								onChange={(e) => setNextPurchase(KINDA_SOON)}
+								className="hidden"
+							/>
+							{nextPurchase === KINDA_SOON ? (
+								<FontAwesomeIcon icon={faCircleDot} className={radioStyle} />
+							) : (
+								<FontAwesomeIcon icon={faCircle} className={radioStyle} />
+							)}
+							Kinda Soon
+						</label>
+
+						<label htmlFor="notSoon" className={radioLabelStyle}>
+							<input
+								type="radio"
+								id="notSoon"
+								name="purchase-frequency"
+								checked={nextPurchase === NOT_SOON}
+								onChange={(e) => setNextPurchase(NOT_SOON)}
+								className="hidden"
+							/>
+							{nextPurchase === NOT_SOON ? (
+								<FontAwesomeIcon icon={faCircleDot} className={radioStyle} />
+							) : (
+								<FontAwesomeIcon icon={faCircle} className={radioStyle} />
+							)}
+							Not Soon
+						</label>
+					</fieldset>
+
+					<button
+						type="submit"
+						className="bg-white text-main rounded-full p-2 w-44 mt-10 shadow-[0_4px_4px_rgba(0,0,0,0.4)] hover:scale-105"
+					>
+						Add Item
+					</button>
+					{submissionConfirmation && (
+						<p className="italic mt-5 text-center max-w-xs">
+							{submissionConfirmation}
```

Comment:
As mentioned, the confirmation message is not easily readable... maybe add a div with a background colour that contrasts nicely against the white text colour.

Reply:
I added a backdrop blur and div color... How does this look?

Reply:
Yes, I love it! Tested it on my device and looks much better!